client/src/popups/setupValidationJobHooks.ts

```diff
@@ -66,5 +68,17 @@ function updateValidationStatusItem(
     : notification.displayText;
   statusItem.busy = false;
 
+  if (!notification.validationRun && notification.errorFile !== undefined) {
+    statusItem.command = {
+      title: "Open file",
+      command: "vscode.open",
+      arguments: [
+        path.join(notification.projectBasePath, notification.errorFile),
```

Comment:
The navigation fails on windows and I suspect it is this line.

Reply:
I suspect we need to pass the result of `path.join` to `decodeUriAndRemoveFilePrefix()`